database/prod/queryClauses/baseClause.cpp

```diff
@@ -0,0 +1,15 @@
+#include "baseClause.h"
+
+QString BaseClause::text() const {
+    return QString("\n %1 %2").arg(title(), valueText());
+}
+
+QString BaseClause::title() const {
+    return title_;
+}
+
+BaseClause::~BaseClause() {}
```

Comment:
в .h ещё можно спорить, но в cpp все функции раскрываем
тогда виднее начало-конец функции при чтении по диагонали и виднее что она задумана как пустая // вроду считается что это как с пустым фором и ';' - если скобки остались сверху, то это могло быть случайно
плюс если станет не-пустой, то просто пишешь тело и всё - не надо двиганием скобок заниматься

вроде и куте сейчас поставили стиль с раскрытием (хотя вроде он и так стоял?)

строки бесконечные, экономить не надо)

```suggestion
BaseClause::~BaseClause() {
}
```